infra/test/integration/go.mod

```diff
@@ -3,71 +3,74 @@ module github.com/terraform-google-modules/dynamic-python-webapp/test/integratio
 go 1.18
 
 require (
-	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.3.0
+	github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test v0.5.0
```

Comment:
Thank you for these updates. I was worried about the older versions of packages in the template I copied from (which doesn't appear to have package updates setup?) but I wasn't sure if bumping the packages would break things.

Reply:
👍  We just published v0.5.0 earlier today, but it should be backwards compatible.